docs/conf.py

```diff
@@ -63,12 +63,24 @@
 nitpicky = True
 nitpick_ignore = [
     ("py:class", "numpy.float64"),
+    # see: https://github.com/numpy/numpydoc/issues/275
+    ("py:class", "None.  Remove all items from D."),
+    ("py:class", "a set-like object providing a view on D's items"),
+    ("py:class", "a set-like object providing a view on D's keys"),
+    ("py:class", "v, remove specified key and return the corresponding value."),  # noqa: E501
+    ("py:class", "None.  Update D from dict/iterable E and F."),
+    ("py:class", "an object providing a view on D's values"),
+    ("py:class", "a shallow copy of D"),
```

Comment:
I think these are warnings from the RTD that should be ignored base on the static class rendering file. But they are indeed too many I'm not sure if it's wanted.

Reply:
Not sure, what to do here.  We can't import from `MutableMapping` for `DistributionCollection` without re-implementing, which seems like a waste.  The other option would be to remove the `DistributionCollection` class since it is used only for `repr`/cleanness of API.  @MUCDK this code was pulled from your starting point, so perhaps you can chime in.

Reply:
I'd be fine with keeping it as it is if it's not too much of an issue. @selmanozleyen wdyt?

Reply:
If typing.Dict doesn't work as well I guess using those ignores is a better solution than re-implementation.